get-started/debug.qmd

```diff
@@ -76,28 +86,24 @@ def bad():
     return str(a_missing_variable)
 ```
 
-When a breakpoint is hit, the program will pause and you can them send commands like `break` (i.e., exit) and `continue` to the debugger. `Ctrl+D` will also exit the debugger.
+When Python hits the `breakpoint()`, it will pause and open the debugger in your terminal.
 
+From there, you can run commands like:
 
-### Print statements
-
-A quick and simple way to debug Shiny applications is to add `print()` statements.
-This lets you see the value of different variables, and how they change when you toggle different inputs.
+- `continue`: resume running the app
+- `exit` or <kbd>Ctrl</kbd> + <kbd>D</kbd>: exit the debugger and stop the app
 
-::: callout-warning
-If your Shiny application is running with Shinylive (Python in the browser), and there is not a visible Python console, then error messages will show up in your browser's JavaScript console.
-:::
+This is helpful for figuring out where things go wrong in your code.
 
 
 ### Shiny debug mode
 
-An advanced option for debugging apps is using the `App(..., debug=True)` argument.
-This is not super useful in general, as it requires some knowledge of Shiny's internals.
+An advanced option for debugging [Shiny Core apps](/docs/express-vs-core.qmd) is using the `App(..., debug=True)` argument.
```

Comment:
In Express, it's `shiny.express.app_opts(debug=True)` (or similar, check import statement)